docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,56 @@
+## FIFO datasink
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new datasink that uses `FIFO` (named pipes) to make the backup process faster by streaming data in parallel from XtraBackup to `xbcloud`. 
+
+When taking a backup, you can either save the backup files locally on the same server, stream the backup to different servers, or to object storage. Percona XtraBackup already supports streaming backups using [`xbcloud`](xbcloud-binary-overview.md). 
+
+Streaming backups to object storage using `xbcloud` works as follows:
+
+* XtraBackup spawns multiple copy threads. Each copy thread reads a chunk of data from a specific file.
+
+* Each copy thread writes the chunk of data to a pipe (STDOUT).
+
+* Xbcloud has a read thread that is responsible for reading each chunk of data from STDIN (standard input device). This chunk is uploaded to an object storage using an async request, and a callback is added to an event handler list.
+
+* The event handler thread of xbcloud is responsible for checking when the TCP socket receives the response from the object storage and executing the callback depending on the return (success or failure).
+
+![image](_static/backup-streamed-to-object-storage.png)
+
+The speed of XtraBackup streaming has a limitation of around 1.8 Gbps (billions of bits per second). This happens because only one thread on the XtraBackup side is written to STDOUT simultaneously. This is done via a mutex to ensure a chunk of data is written from the beginning till the end. This architecture works fine when uploading data via WAN. However, for streaming backups using LAN, the most popular setup is 10 Gbps or more.
+
+With FIFO datasink feature, you can benefit from faster backups by streaming data in parallel.
```

Comment:
This should be moved to the first paragraph. Highlite the benefit.

Reply:
updated